monitoring/grafana/provisioning/datasources/datasource.yml

```diff
@@ -0,0 +1,53 @@
+# # config file version
```

Comment:
This file is OK, but i think you should remove the comments to make it cleanner.